### ISSUE TITLE
Initiating a set time from a hidden seconds input

### ISSUE THREAD

**Issue by Braunson** [digaev/jQuery-timeDurationPicker#3]
Any suggestions on the best way initiate an option to preset the time (in seconds) or set an input id/name with the seconds there (i.e like your hidden input in the demo)
1. Set duration in the options array
2. Set duration after the plugin has initialized

These two methods would be awesome to have. I tried using the `defaultValue` option that seems to be currently there but it doesn't work.

**Comment by digaev** [comment]
> Any suggestions on the best way initiate an option to preset the time (in seconds) or set an input id/name with the seconds there (i.e like your hidden input in the demo)

I think the best way would be to use `defaultValue` option with one of possible value types:
1. `Integer` - an initial number of seconds (already done)
2. `function` - a function, which must return an `Integer` value (number of seconds)

> Set duration after the plugin has initialized

I'll see what i can do.

>  I tried using the `defaultValue` option that seems to be currently there but it doesn't work.

How did you do that? This should work:

```
$('#duration').timeDurationPicker({ defaultValue: 123456789 });
```

[jsfiddle](https://jsfiddle.net/kyybuaub/)

**Comment by Braunson** [comment]
@digaev I'm using defaultValue but it doesn't seem to update the placeholder / value, not until I click the input to open the model with dropdowns and click 'Update' then it shows the updated placeholder x hours and x minutes.
